requirements.txt

```diff
@@ -31,3 +31,5 @@ six==1.12.0
 uritemplate==3.0.0
 urllib3==1.24.1
 coverage==4.5.3
+memcached==1.4.15
```

Comment:
This line breaks travis. Possibly uneeded.

Reply:
A good tip is to: `pip install package` .
Then later if you are happy with the package: `pip freeze > requirements.txt`